### DISCUSSION TITLE
How to fix this PL/SQL Anonymous Block Color?

### DISCUSSION THREAD

**Discussion by vm7md98** [dracula/dracula-theme#724] [Q&A] [answered]
Just to mention this, i'm using sqlDeveloper from Oracle.

As it shown in the pictures, dracula theme works fine  when I'm not ending the query with semicolon ";".
![1](https://user-images.githubusercontent.com/56771415/172140424-bc6e1d9f-5810-4177-8a4f-869afb586540.PNG)

But I have to use the semicolon to run the query and once I type ";", it sets the query to white.
![2](https://user-images.githubusercontent.com/56771415/172140453-05df15d1-c4d1-45bf-9fa0-5a8d78fa0161.PNG)

Any ideas to fix this? Maybe a tweak in the XML file?

**Comment by luxonauta** [comment, marked as answer]
Thanks for providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.
